### ISSUE TITLE
Possibility to deactivate "street name", "speed" in navigation view with code, like UI Elements in normal view with setDisabledIds method. 

### ISSUE THREAD

**Issue by pTCNX** [osmandapp/OsmAnd#9981]
Is it possible, to deactivate UI Elements like "street name" and "speed" in the navigation view with code?

**Comment by vshcherb** [comment]
Of course, it's possible to do with Profile but it is not possible to do with setDisabledIds. 
It is possible to extend the methods though it's recommended to use profile settings in such cases.
